buildkite/src/Command/Base.dhall

```diff
@@ -58,6 +60,7 @@ let Config =
   { Type =
       { commands : List Cmd.Type
       , depends_on : List TaggedKey.Type
+      , artifact_paths : List Text
```

Comment:
probably makes sense to use something like List `SelectFiles.Type` in order to capture glob patterns